### PR TITLE
Order - improve the required fields

### PR DESCRIPTION
**Pull request by GabrielBlondeau-Kirby**
Remove the `Required` label on a few fields.

### CODE REVIEW

File: source/includes/_orders.md [hunk 1]
```diff
@@ -168,11 +169,11 @@ Creates a new order.
 Argument | Type | Description
 -------- | ---- | -------
 **user** | `object` | Id of the [`user`](#users) who's creating the order
```
Comment (KristopherJordan) [inline source/includes/_orders.md]: We don't need the `user`field anymore when creating an order

File: source/includes/_orders.md [hunk 2]
```diff
@@ -10,24 +10,25 @@ The `total_amount` is set by the `items` and the `resources`. See section below.
...
-resources | `array` | An array of [`resources`](#resources) associated with the order.
+human_id | `string` | Human readable ID that identifies the order easily. e.g. `3AG7UA`
+note | `string` | Field used to send notes between an user and a provider
+order_status | `string` | Status of the Order _default is CREATED, additionally there are: PROCESSING, SUCCESS, CANCELLED and FAILED_
+payments | `array` | List of [`Payment`](#payments) objects associated with order
+payment_method | `object` | [`Payment Methods`](#payment_methods)
+provider | `string` | Provider assigned to an order used to pay the order
 units | `number` | Number of unique products in an order
-total_quantity | `number` | Total number of products in an order
+user | `object` | [`User`](#users) associated with the order
```
Comment (KristopherJordan) [inline source/includes/_orders.md]: You don't need the user

Comment (GabrielBlondeau-Kirby) [reply source/includes/_orders.md]: Hmm :thinking: it's just the description of what an Order model contains

Comment (GabrielBlondeau-Kirby) [reply source/includes/_orders.md]: But you are right ;)

File: source/includes/_orders.md [hunk 3]
```diff
@@ -10,24 +10,24 @@ The `total_amount` is set by the `items` and the `resources`. See section below.
...
+delivery_status | `string` | Status of the delivery _default is PENDING, additionally there are: ACCEPTED, REJECTED, ON_THE_WAY, ARRIVED, DONE and READY_FOR_DELIVERY_
+delivery_time | `object` | Expected arrival time for delivery, `timestamp` format
 items | `array` | List of items associated with an order
-resources | `array` | An array of [`resources`](#resources) associated with the order.
+human_id | `string` | Human readable ID that identifies the order easily. e.g. `3AG7UA`
+note | `string` | Field used to send notes between an user and a provider
+order_status | `string` | Status of the Order _default is CREATED, additionally there are: PROCESSING, SUCCESS, CANCELLED and FAILED_
+payments | `array` | List of [`Payment`](#payments) objects associated with order
+payment_method | `object` | [`Payment Methods`](#payment_methods)
+provider | `string` | Provider assigned to an order used to pay the order
```
Comment (KristopherJordan) [inline source/includes/_orders.md]: Please change this back to `Provider assigned to an order`

**Comment by GabrielBlondeau-Kirby** [comment]
@KristopherJordan Done

**Pull request merged**